assets/src/edit-story/components/canvas/useCanvasKeys.js

```diff
@@ -117,6 +122,15 @@ function useCanvasKeys(ref) {
   ]);
   useGlobalKeyDownEffect('esc', () => clearSelection(), [clearSelection]);
 
+  useGlobalKeyDownEffect(
+    { key: ['mod+a'] },
+    () => {
+      const elementIds = currentPage.elements.map(({ id }) => id);
+      setSelectedElementsById({ elementIds });
+    },
+    [clearSelection, currentPage, setSelectedElementsById]
```

Comment:
What's `clearSelection` doing here? Copy-paste error?

Reply:
Good catch. That is so odd the linter didn't catch the extra dep.

Reply:
@carlos-kelly is this behavior different for built-in hooks?

Reply:
My guess is that `useGlobalKeyDownEffect` is backed by `useEffect` since it's things like `useCallback` or `useMemo` that complain with extra deps.
<img width="463" alt="Screen Shot 2020-07-31 at 8 11 19 AM" src="https://user-images.githubusercontent.com/1738349/89038160-6de14b00-d305-11ea-9ae2-8d72ad64661d.png">